File: frontend/lib/models/admin_mechanic_account_change_dto.dart
class AdminMechanicAccountChangeDto {
  final String? accountTypeName;
  final String? accessLevelName;
  final int? clubId;
  final bool? attachToClub;

  const AdminMechanicAccountChangeDto({
    this.accountTypeName,
    this.accessLevelName,
    this.clubId,
    this.attachToClub,
  });

  Map<String, dynamic> toJson() => {
        'accountTypeName': accountTypeName,
        'accessLevelName': accessLevelName,
        'clubId': clubId,
        'attachToClub': attachToClub,
      };
}

File: backend/CoreService/src/main/java/ru/bowling/bowlingapp/DTO/AdminMechanicAccountChangeDTO.java
package ru.bowling.bowlingapp.DTO;

import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@NoArgsConstructor
@AllArgsConstructor
public class AdminMechanicAccountChangeDTO {
    private String accountTypeName;
    private String accessLevelName;
    private Long clubId;
    private Boolean attachToClub;
}
